sentry-android-core/src/main/java/io/sentry/android/core/ScreenshotEventProcessor.java

```diff
@@ -33,6 +34,8 @@
 public final class ScreenshotEventProcessor
     implements EventProcessor, Application.ActivityLifecycleCallbacks, Closeable {
 
+  private static @NotNull ScreenshotEventProcessor instance;
```

Comment:
H: if `createInstance` was not called, this returns null, so it should be nullable or we should not use a singleton pattern